packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -165,11 +165,13 @@ module.exports = {
       }
     ]
   },
+  // @remove-on-eject-begin
   // Point ESLint to our predefined config.
   eslint: {
-    configFile: path.join(__dirname, 'eslint.js'),
+    configFile: path.join(__dirname, '../.eslintrc'),
     useEslintrc: false
   },
+  // @remove-on-eject-end
```

Comment:
I didn't mean ejected, I meant `webpack.config.prod.js`. Don't we need to update path in it as well as `.dev.js`?